gyp_xwalk

```diff
@@ -351,6 +351,9 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if sys.platform == 'win32':
```

Comment:
I'd like to go back to this. I've just done a quick test here, adding `'enable_widevine%': 1` to the Linux block and everything worked as expected (ie. building the `widevinecdm` target built the stub files, and `-DCDM_IMPLEMENTATION` was passed to the compiler). If you followed what I typed above, it probably failed because I wrote `'foo%: 1'` instead of `'foo%': 1`.